### PR TITLE
Sarahmood/issue66

### LINKED ISSUE #66: HOTFIX: Enable all CORS for the frontend to use the API
It looks like the front end team is having a problem with fetching from our API with a CORS error, using the `cors` package we should enable requests coming from all cross origin requests

### PR DESCRIPTION
**Pull request by sarahmood**
**add cors package to dev branch**

### CODE REVIEW

File: src/app.js
Comment (muhammed-shihebi) [inline src/app.js:32]: @sarahmood, this will work, I tested it, but it will allow any origin (anyone on the web can use our API) but we want only the front-end app to be able to use our API. 

From the [CORS package](https://www.npmjs.com/package/cors#simple-usage-enable-all-cors-requests), we can do this by replacing this code with the following: 

```
const whitelist = [
    'http://localhost:3000'
];
const corsOptions = {
    origin: function (origin, callback) {
        if (whitelist.indexOf(origin) !== -1) {
            callback(null, true);
        } else {
            callback(new Error('Not allowed by CORS'));
        }
    },
};
app.use(cors(corsOptions));
```

**Review by muhammed-shihebi — APPROVED** [review]
Hi @sarahmood, thank you for your quick response on this issue. I am going to approve this so you don't have to wait for the approval again, but please check out the comment I left.

File: src/app.js
Comment (sarahmood) [reply src/app.js:32]: @muhammed-shihebi  Thank you Nur! I actually doubt that and i asked on slack but i  got no respond! I will change it now.

**Review by halitbatur — APPROVED** [review]

**Pull request merged**
